tests/e2e/tests_suite_test.go

```diff
@@ -111,7 +112,9 @@ func TestTests(t *testing.T) {
 	RegisterFailHandler(Fail)
 	fmt.Fprintf(GinkgoWriter, "Starting test suite\n")
 	RunSpecs(t, "Tests")
-	Cleanup()
+	if (ginkgoutil.FailureBehaviourEnvSwitcher{}).IsStopOnFailure() {
```

Comment:
Should not cleanup when stop on failure is enabled.
```suggestion
	if (!ginkgoutil.FailureBehaviourEnvSwitcher{}).IsStopOnFailure() {
```